src/objectTranslations/Schedulable__c-de.objectTranslation

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <caseType>Nominative</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Nominative</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Accusative</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Accusative</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Genitive</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Genitive</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Dative</caseType>
+        <plural>false</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <caseValues>
+        <caseType>Dative</caseType>
+        <plural>true</plural>
+        <value><!-- Schedulable --></value>
+    </caseValues>
+    <fields>
+        <help>Legt fest, ob der Auftrag aktiv ist</help>
+        <label>Aktiv</label>
+        <name>Active__c</name>
+    </fields>
+    <fields>
+        <help>Klasse, deren Ausführung mit einer bestimmten Häufigkeit geplant wird</help>
+        <label>Klassenname</label>
+        <name>Class_Name__c</name>
+    </fields>
+    <fields>
+        <help>Wie oft der Auftrag ausgeführt wird</help>
+        <label>Häufigkeit</label>
+        <name>Frequency__c</name>
+        <picklistValues>
+            <masterLabel>Daily</masterLabel>
+            <translation><!-- Daily --></translation>
```

Comment:
@dospromptman remind me why we don't want these picklist value for Frequency__c translated?  Is it because our code isn't looking at the developer name (masterLabel) internally?  do we have a backlog item related to this?